src/components/AuthForm.js

```diff
@@ -1,7 +1,38 @@
 import React from "react";
 import InputForm from "./Form";
+import {useState} from "react";
+import axios from 'axios';
+
+//let axios = require('axios');
 
 function AuthForm (props) {
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  async function submitData () {
+    const data = {
+      "user": {
+      username,
+      email,
+      password
+      }
+    }
+
+    let register = await axios.post('https://launchup-prisma.herokuapp.com/api/users', data)
```

Comment:
Its better to have the endpoint saved as a global environment variable rather than hard coding . That way if it changes minimal work is needed. 
The route itself should be coded in here (i.e /users)
Have a look at this article for more info.
https://adostes.medium.com/using-environment-variables-in-a-react-application-ac3b6c307373

Reply:
Addressed. Also, an .env file has been created and the main API url (https://launchup-prisma.herokuapp.com/api) has been added to the env file. Thanks!